src/conversions.jl

```diff
@@ -271,12 +270,22 @@ cnvt(::Type{HSI{T}}, c::Color3) where {T} = cnvt(HSI{T}, convert(RGB{T}, c))
 # Everything to XYZ
 # -----------------
 
-function invert_rgb_compand(v)
-    v <= 0.04045 ? v/12.92 : ((v+0.055) /1.055)^2.4
+function invert_srgb_compand(v::Fractional)
+    v <= 0.04045 && return v/12.92
+    # the following is an optimization technique for `((v+0.055) /1.055)^2.4`.
+    # see also: srgb_compand(v::Fractional)
+    x = (v + 0.055) / 1.055
+    return x^2 * exp(0.4 * log(x)) # 2.4 == 2 + 0.4
+end
+
+const invert_srgb_compand_n0f8 = [invert_srgb_compand(v/255) for v = 0:255] # LUT
+
+function invert_srgb_compand(v::N0f8)
+    invert_srgb_compand_n0f8[v.i + 1]
```

Comment:
```suggestion
    invert_srgb_compand_n0f8[retinterpret(UInt8, v) + 1]
```
I know it's longer, but this uses an "official" interface rather than digging into the struct.

Entirely optional, you can leave it like it is.

Reply:
Oh, and pre-calculating this seems like a good idea!